instrumentation/active_record/lib/opentelemetry/instrumentation/active_record/patches/persistence.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveRecord
+      module Patches
+        # Module to prepend to ActiveRecord::Persistence for instrumentation
+        module Persistence
+          def save!(**options, &block)
+            tracer.in_span("#{self.class}#save!") do
+              super
+            end
+          end
+
+          def save(**options, &block)
+            tracer.in_span("#{self.class}#save") do
+              super
+            end
+          end
+
+          def delete
+            tracer.in_span("#{self.class}#delete") do
+              super
+            end
+          end
+
+          def destroy!
+            tracer.in_span("#{self.class}#destroy!") do
+              super
+            end
+          end
+
+          def destroy
+            tracer.in_span("#{self.class}#destroy") do
+              super
+            end
+          end
+
+          def update_attribute(name, value)
+            tracer.in_span("#{self.class}#update_attribute") do
+              super
+            end
+          end
+
+          def update!(attributes)
+            tracer.in_span("#{self.class}#update!") do
```

Comment:
Here as well in some extra methods here, can we record the attributes or other information that we have at this stage?

Reply:
This is something I would like to separate from this initial PR.  We don't currently have any semantic conventions around ORMs. 

I'm also hesitant adding all attributes to the span in say a call to `create`, it has PII implications that may prevent some early adoption of this instrumentation.

That's not to say we cannot do this eventually, I'd just prefer to do small iterations.